asf_core_data/pipeline/mcs/generate_mcs_data.py

```diff
@@ -64,13 +71,32 @@ def concatenate_save_raw_installations():
             end_date = datetime.date(year, end_month + 1, 1) - datetime.timedelta(
                 days=1
             )
+
+            installations_key = " ".join(
+                [
+                    key
+                    for key in (list(key_and_df[1].keys()))
+                    if "installs" in key.lower()
+                ]
+            )
+            installer_key = " ".join(
+                [key for key in (list(df.keys())) if "installer" in key.lower()]
+            )
```

Comment:
hmmm odd that they're empty - because the new dataset returned is a dictionary where the key is the tab name and the value is the pd.DataFrame, this is supposed to get the installations key and the installer key to be able to call the dataframe. I'll take a second look!

Reply:
okay i see the problem in the end - it should be key_and_df[1].keys() instead of df - this then returns the keys for the tab name and the pd.DataFrame.